.github/actionlint.yaml

```
@@ -1,6 +1,6 @@
 self-hosted-runner:
   # Pending https://github.com/rhysd/actionlint/issues/533
-  labels: ["windows-11-arm"]
+  labels: ["windows-11-arm", "macos-15-intel"]
```

This is in fact no longer pending: https://github.com/rhysd/actionlint/pull/542 / https://github.com/rhysd/actionlint/pull/572
```suggestion
  labels: []
```

---

so maybe drop the comment above too?

---

No, it's been merged but still pending release.

https://github.com/rhysd/actionlint/releases